spec/controllers/hyrax/registrations_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe Hyrax::RegistrationsController, type: :controller do
+  routes { Rails.application.routes }
+
+  before do
+    allow(Flipflop).to receive(:account_signup?).and_return(account_signup_enabled)
+    # Recommended by Devise: https://github.com/plataformatec/devise/wiki/How-To:-Test-controllers-with-Rails-3-and-4-%28and-RSpec%29
+    @request.env['devise.mapping'] = Devise.mappings[:user]
```

Comment:
Ah, this is if you are not using https://github.com/samvera-labs/hyrax/blob/8d44caab1ec373a82fe7b079fb670193263d6308/spec/spec_helper.rb#L211